db/create_tables.sql

```diff
@@ -55,6 +55,8 @@ CREATE TABLE IF NOT EXISTS `state_machines` (
   `id` BIGINT(20) NOT NULL AUTO_INCREMENT,
   `name` VARCHAR(255) NOT NULL,
   `version` TINYINT UNSIGNED NOT NULL,
+  `start_state_id` BIGINT(20) NOT NULL,
+  `description` VARCHAR(300) DEFAULT NULL,
   `created_at` DATETIME(3) DEFAULT NULL,
   `updated_at` DATETIME(3) DEFAULT NULL,
```

Comment:
It keeps in this format "2015-08-31 15:22:25.535". I think having 3 digits for fractional seconds is good enough.